lib/charms/operator_libs_linux/v0/systemd.py

```diff
@@ -56,11 +56,15 @@
 LIBID = "045b0d179f6b4514a8bb9b48aee9ebaf"
 
 # Increment this major API version when introducing breaking changes
-LIBAPI = 0
+LIBAPI = 1
```

Comment:
With this change, the library also needs to be put in the `lib/charms/operrator_libs_linux/v1` directory